db/sql/blocks/getIdSequence.sql

```diff
@@ -6,7 +6,7 @@
 
 WITH current_round AS
   (
-    SELECT CEIL(b.height / ${delegates}::float)::bigint
+    SELECT ceil(b.height / ${delegates}::float)::bigint
```

Comment:
Although it works, is there any specific reason to change `CEIL` to down case?

Reply:
Actually, I changed all standard SQL functions to low case, not just here. The convention suggests - you use upper case only for SQL key words + CONSTANTS, but not for function names :wink:

Reply:
👍